app/src/main/res/values/strings.xml

```diff
@@ -477,4 +477,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_religion_bahai">Bahá\'í Faith</string>
     <string name="quest_religion_caodaist">Caodaism</string>
     <string name="quest_religion_chinese_folk">Chinese folk religion</string>
+    <string name="quest_religion_for_wayside_shrine_title">What religion is represented at this shrine?</string>
+    <string name="quest_religion_confucian">Confucianism</string>
```

Comment:
I think it really should be chinese folk instead of confucian.

Reply:
OK, even if Confucian shrines exists nobody bothered to tag them.